### PR TITLE
bug 1677623 added openshift_release pre-install descriptions

### PR DESCRIPTION
**Pull request by sheriff-rh**
Duplicate of #13865 due to commit errors, which is now closed. 

BZ:
https://bugzilla.redhat.com/show_bug.cgi?id=1677623

This PR is for 3.9 and below (to 3.4), adding a descriptor for `openshift_release in the `pre-install checks` table, which is in two locations.

@wmengRH PTAL, thanks.

### CODE REVIEW

**Comment by wmengRH** [comment]
LGTM. 
Thanks.

File: install/configuring_inventory_file.adoc [hunk 1]
```diff
@@ -852,6 +852,9 @@ Use the Device Mapper Storage Driver guide].
 available either locally or in at least one of the configured container image
 registries on the host machine.
 
+|`openshift_release`
+|Specifies the generic release of {product-title} for containerized installations. For RPM installations, refer to `package_availability`.
```
Comment (kalexand-rh) [inline install/configuring_inventory_file.adoc]: Do you mean that for RPM installations you need to use `package_availability` instead? If so, I'd do something like s/refer to \`package_availability\`/set a \`package_availability\` value   (two places)

**Review by kalexand-rh — COMMENTED** [review]
One question. Otherwise LGTM.

File: install/configuring_inventory_file.adoc (hunk 1 above)
Comment (sheriff-rh) [reply install/configuring_inventory_file.adoc]: I agree, thank you @kalexand-rh !

**Pull request closed without merging**